apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js

```diff
@@ -49,68 +47,80 @@ function WpcomNux() {
 
 	const dismissWpcomNux = () => setWpcomNuxStatus( { isNuxEnabled: false } );
 
+	// @TODO: Change to 'Welcome to your new website' for Gutenboarding, in other situations keep "editor"
+	const welcomeHeading = __( 'Welcome to the WordPress editor' );
```

Comment:
@Addison-Stavlo hi! Do you know how I could detect `?is-gutenboarding` status the best way here?

Reply:
From FSE/Block-editor side of the iframe Addison added this function: https://github.com/Automattic/wp-calypso/blob/master/apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js#L697

Then any override is being done by checking the `calypsoifyGutenberg.isGutenboarding` global variable with some [hook fallback](https://github.com/Automattic/wp-calypso/blob/master/apps/wpcom-block-editor/src/calypso/features/gutenboarding-editor-overrides.js#L19-L21).